lib/js/src/manager/permission/_PermissionFilter.js

```diff
@@ -35,10 +35,11 @@ import { uuid } from '../../util/uuid.js';
 /**
```

Comment:
Missed spots for changing to _PermissionFilter for the set methods:

`@returns {PermissionFilter}`

Reply:
docs are now updated